Luchynska Tetiana/my-todo-list/src/app/todo/todo.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { element } from 'protractor';
+
+@Component({
+  selector: 'app-todo',
+  templateUrl: './todo.component.html',
+  styleUrls: ['./todo.component.css']
+})
+export class TodoComponent implements OnInit {
+
+  allowAddingItem = false;
+  todo = [{id: 0, label: "Drink coffee", done: false, editing: false},{id: 1, label: "Sleep", done: false, editing: false}, {id: 2, label: "Do TODO list", done: false, editing: false}];
+  nextId = 3;
+  constructor() { }
+
+  ngOnInit(): void {
+
+  }
+
+  onUpdateInput(event: Event){
+    if((<HTMLInputElement>event.target).value==='') this.allowAddingItem = false;
```

Comment:
It's better to use brackets for each condition to avoid mistakes in the future.

Reply:
fixed